pkg/cmd/admin/policy/cani.go

```diff
@@ -209,13 +226,18 @@ func (o *canIOptions) listAllPermissions() error {
 		if err != nil {
 			return err
 		}
-		rulesReviewStatus = whatCanYouDo.Status
 
+		policyRules = whatCanYouDo.Status.Rules
+		rulesReviewResult = whatCanYouDo
+	}
+
+	if o.Printer != nil {
+		return o.Printer.PrintObj(rulesReviewResult, o.Out)
 	}
 
 	writer := tabwriter.NewWriter(o.Out, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
```

Comment:
Would it make sense to get rid of this+4 lines and always rely on the printer, with the HumanReadablePrinter by default?

> I agree makes sense, though I think we might have to [add a handler](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/printer.go#L79) here with the current printing behavior in [DescribePolicyRule](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L1300)

https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/printer.go#L61